v_4_0_0/cloudconfig.go

```diff
@@ -14,7 +14,7 @@ import (
 
 const (
 	defaultRegistryDomain = "quay.io"
-	kubernetesImage       = "giantswarm/hyperkube:v1.12.3"
+	kubernetesImage       = "giantswarm/hyperkube:v1.13.0"
```

Comment:
in retagger we have the `v1.13.1` https://github.com/giantswarm/retagger/blob/master/images.yaml#L114

```suggestion
	kubernetesImage       = "giantswarm/hyperkube:v1.13.1"
```